dockers/docker-orchagent/enable_counters.py

```diff
@@ -14,6 +14,8 @@ def enable_counters():
     enable_counter_group(db, 'PORT')
     enable_counter_group(db, 'QUEUE')
     enable_counter_group(db, 'PFCWD')
+    enable_counter_group(db, 'PG_WATERMARK')
+    enable_counter_group(db, 'QUEUE_WATERMARK')
```

Comment:
Once the corresponding FLEX_COUNTER_GROUP_TABLE is enabled, CONFIG_DB will have the entry flow into it even if the saved config may not have the entry enlisted. In the warm-reboot case, Orch::bake() can detect the entry, and will populate the entry into m_toSync. The counter poll postponement will thus lose its purpose in the warm-reboot case. @pavel-shirshov